pkg/opensearch/client/client.go

```diff
@@ -108,14 +109,14 @@ func NewClient(ctx context.Context, ds *backend.DataSourceInstanceSettings, http
 
 	version, err := ExtractVersion(jsonData.Get("version"))
 	if err != nil {
-		return nil, fmt.Errorf("version is required, err=%v", err)
+		return nil, errorsource.DownstreamError(fmt.Errorf("version is required, err=%v", err), false)
```

Comment:
Could you use `backend.DownstreamError()` we are in process of deprecating `experimental/errorsource` and we won't have to fix this later.

Reply:
Whoops! Working on autopilot, thanks 😊